adal/build.gradle

```diff
@@ -250,8 +250,8 @@ publishing {
             name "vsts-maven-adal-android"
             url "https://identitydivision.pkgs.visualstudio.com/_packaging/AndroidADAL/maven/v1"
             credentials {
-                username project.findProperty("vstsUsername")
-                password project.findProperty("vstsMavenAccessToken")
+                username System.getenv("VSTS_ENV_USER_NAME") != null ? System.getenv("VSTS_ENV_USER_NAME") : project.findProperty("vstsUsername")
```

Comment:
How do you feel about giving this a 'schema', in case more artifact streams are added later?

As an idea:
`ENV_<host>_<stream_type>_<stream_name>_<credential_type>`

In this case, something like:
`ENV_VSTS_MVN_ANDROIDADAL_USERNAME`
`ENV_VSTS_MVN_ANDROIDADAL_ACCESSTOKEN`

Reply:
Good suggestion @iambmelt 👍